master/buildbot/test/unit/test_process_properties.py

```diff
@@ -990,9 +990,6 @@ def testUpdateFromPropertiesNoRuntime(self):
         self.assertEqual(self.props.getPropertySource('x'), 'old')
 
     def test_setProperty_notJsonable(self):
-        with self.assertRaises(TypeError):
```

Comment:
What happens if we set non-jsonable renderable? I think we should have a test that verifies the effect in that case.

Reply:
There will be an exception during the build.
The point of this check was to raise an exception as soon as possible to ease the debug, but now we accept renderable, the check inside db api is done:

https://github.com/buildbot/buildbot/blob/4ef65c3ab815db1b997cf359c74b0ab1185a3d68/master/buildbot/db/builds.py#L205